dogs.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>html200-adopt-a-dog</title>
+      <link rel="stylesheet" href="./styles/styles.css">
+  </head>
+  <body>
+
+
+    <header class="header-images">
+      <div class="background2Box">
+      <img class="logoHeader" src="./images/adoptadog-logo.svg" alt="Company Logo">
+      <h2 class="headertext"> Seattle's premiere dog adoption agency</h2>
+    </div>
+      <a class="checkout "href="./checkout.html">Checkout</a>
+    </header>
+
+    <nav>
+      <ul>
+        <li>
+          <a href="./index.html">Home</a>
+        </li>
+        <li>
+          <a href="#">Dogs</a>
+        </li>
+        <li>
+          <a href="./blog.html">Blog</a>
+        </li>
+      </ul>
+    </nav>
+
+    <main id="wrapper">
+      <section class="flex-container">
+        <h2 class="div1">Here are all of our Dogs!</h2>
+          <article class="div2">
```

Comment:
I'd argue these aren't articles. A div is fine. Also just a reminder to use semantic class names